example/ck_tile/01_fmha/fmha_fwd.cpp

```diff
@@ -51,10 +51,15 @@ auto create_args(int argc, char* argv[])
         .insert("s_k", "0", "seqlen_k, 0 means equal to s")
         .insert("d", "128", "head dim for q, k")
         .insert("d_v", "0", "head dim for v, 0 means equal to d")
-        .insert("scale", "0", "scale factor. 0 means equal to 1/sqrt(hdim)")
-        .insert("descale_q", "1", "scale factor for fp8 quantization")
-        .insert("descale_k", "1", "scale factor for fp8 quantization")
-        .insert("descale_v", "1", "scale factor for fp8 quantization")
+        .insert("scale", "0", "scale factor of S. 0 means equal to 1/sqrt(hdim)")
+        .insert("scale_p",
+                "0",
+                "scale factor of P(output of softmax), only works in f8 static quantization."
```

Comment:
replace `f8` to `fp8` here.

Reply:
1bd814b1ff9189f48369fb2d68aa2f58884706ab